lua/nvim-web-devicons/icons-default.lua

```diff
@@ -1136,6 +1136,12 @@ local icons_by_file_extension = {
     cterm_color = "28",
     name = "D",
   },
+  ["d.ts"] = {
+    icon = "󰛦",
```

Comment:
This one's material design, the other javascript icons are seti.

Please change to seti for consistency.

You can use https://www.nerdfonts.com/cheat-sheet and paste the icon into the search box.